tests/behat/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "require-dev": {
+        "sensiolabs/behat-page-object-extension": "^2.1",
+        "jarnaiz/behat-junit-formatter": "^1.3",
+        "Drupal/Drupal-Extension": "^3.3",
+        "emuse/behat-html-formatter": "^0.1.0"
+    },
+    "config": {
+        "bin-dir": "bin/"
+    },
+    "require": {
+        "behat/behat": "^3.4",
+        "behat/mink": "^1.4@stable",
+        "behat/mink-extension": "^2.2",
+        "behat/mink-goutte-driver": "^1.2",
+        "behat/mink-selenium2-driver": "^1.3",
+        "curl/curl": "^1.8",
+        "michaelc/intdiv-compat": "^1.0",
+        "paragonie/random_compat": "^2.0"
+    }
+}
```

Comment:
The file composer.lock should also be added to the repository to ensure that all test and developer are using the exact same version of behat.

Reply:
Uploaded.